drivers/tapdisk-nbdserver.c

```diff
@@ -1131,7 +1131,7 @@ tapdisk_nbdserver_newclient_unix(event_id_t id, char mode, void *data)
 
 	INFO("server: got connection\n");
```

Comment:
In this case it will always be `new fixed` as the socket only supports that. When starting an NBD server via passing an fd (as is done by SXM) it goes through an entirely different path.